migrating_to_2.0/recipes.rst

```diff
@@ -27,7 +27,7 @@ Python import statements
    :caption: **To:**
 
     from conan import ConanFile
-    from conan.tools.files import save, load
+    from conan.tools.files import get, save, load
```

Comment:
This doc doesn't aim to be an exhaustive list of all possible imports, it is fine to add this, but not necessary at all.